### PR TITLE
docs(collapse): decompose Basic demo

### LINKED ISSUE #3789: docs(collapse): decompose Basic demo
Decompose to: 
- Basic _(remove the excess: collapsed, expanded)_
- Collapsed & expanded events listening _(add demo for showing/hiding event)_

related to #3238

### PR DESCRIPTION
**Pull request by artolshansky**
close #3789

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+collapse/collapse-section.list.ts [hunk 1]
```diff
@@ -29,6 +30,13 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/basic/basic.html'),
         outlet: CollapseDemoComponent
       },
+      {
+        title: 'Collapsed & expanded events listening',
```
Comment (IlyaSurmay) [inline demo/src/app/components/+collapse/collapse-section.list.ts]: just `Events`

File: demo/src/app/components/+collapse/collapse-section.list.ts [hunk 2]
```diff
@@ -29,6 +30,13 @@ export const demoComponentContent: ContentSection[] = [
...
+        anchor: 'events',
+        component: require('!!raw-loader?lang=typescript!./demos/events/events'),
+        html: require('!!raw-loader?lang=markup!./demos/events/events.html'),
+        outlet: CollapseDemoEventsComponent
```
Comment (IlyaSurmay) [inline demo/src/app/components/+collapse/collapse-section.list.ts]: add a description field with text `Collapse directive exposes 2 events: <code>collapsed</code>, that fires when a content was hidden, and  <code>expanded</code>, that fires when a content was shown`

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Review by valorkin — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Needs to fix conflicts

**Comment by valorkin** [comment]
@olshansky why close PR and delete branch?

**Comment by artolshansky** [comment]
@valorkin mistake 
open same pr #3977

**Pull request closed without merging**